fuse_examples/tests/conf/config.yaml

```diff
@@ -0,0 +1,55 @@
+paths:
+  working_dir : .
+  model_dir : ${paths.working_dir}/model_dir
+  inference_dir : ${paths.working_dir}/model_new/infer_dir
+  eval_dir : ${paths.working_dir}/model_new/eval_dir
+  cache_dir : ${paths.working_dir}/cache/PICAI
+  clinical_file : ${oc.env:PICAI_DATA_PATH}/labels/picai_labels-main/clinical_information/marksheet.csv
+  data_misc_dir : ${paths.working_dir}/data_misc
+  data_split_filename:  split.pkl
+  data_dir :  ${oc.env:PICAI_DATA_PATH}/images
+  seg_dir : ${oc.env:PICAI_DATA_PATH}/labels/picai_labels-main/csPCa_lesion_delineations/human_expert/resampled
+run: 
+  running_modes : ['train', 'infer', 'eval']
+train:
+  target : seg3d
+  series_config : ['_t2w']
+  resize_resolution : [23,384,384]
+  reset_cache: False
+  num_workers : 16
+  num_folds : 5
+  train_folds : [0,1,2,3]
+  validation_folds : [4]
+  batch_size: 16
+  learning_rate : 1e-3
+  weight_decay :  0
+  resume_checkpoint_filename : 
+  trainer:
+    accelerator : gpu
+    devices : 1
+    num_epochs : 100
```

Comment:
you do 100 epochs just for a unit test?

Reply:
I actually override it in the test,
not sure what is the better way to do it,
@mosheraboh any idea?